src/test/fuzz/feefrac.cpp

```diff
@@ -59,23 +48,14 @@ std::strong_ordering MulCompare(int64_t a1, int64_t a2, int64_t b1, int64_t b2)
     int sign_b = (b1 == 0 ? 0 : b1 < 0 ? -1 : 1) * (b2 == 0 ? 0 : b2 < 0 ? -1 : 1);
     if (sign_a != sign_b) return sign_a <=> sign_b;
 
-    // Compute absolute values.
-    uint64_t abs_a1 = static_cast<uint64_t>(a1), abs_a2 = static_cast<uint64_t>(a2);
-    uint64_t abs_b1 = static_cast<uint64_t>(b1), abs_b2 = static_cast<uint64_t>(b2);
-    // Use (~x + 1) instead of the equivalent (-x) to silence the linter; mod 2^64 behavior is
-    // intentional here.
-    if (a1 < 0) abs_a1 = ~abs_a1 + 1;
-    if (a2 < 0) abs_a2 = ~abs_a2 + 1;
-    if (b1 < 0) abs_b1 = ~abs_b1 + 1;
-    if (b2 < 0) abs_b2 = ~abs_b2 + 1;
+    // Compute absolute values of products.
```

Comment:
Is it important to distinguish the 0 values from the positive ones when comparing the signs?
If not, maybe we can simplify a bit:
```C++
    // Compute and compare signs.
    int sign_a = (a1 < 0 ? -1 : 1) * (a2 < 0 ? -1 : 1);
    int sign_b = (b1 < 0 ? -1 : 1) * (b2 < 0 ? -1 : 1);
    if (sign_a != sign_b) return sign_a <=> sign_b;

    auto result = Abs256(a1) * Abs256(a2) <=> Abs256(b1) * Abs256(b2);
    return (sign_a >= 0) ? result : 0 <=> result;

Reply:
In that case consider only the second part of the suggestion:
```C++
    auto result = Abs256(a1) * Abs256(a2) <=> Abs256(b1) * Abs256(b2);
    return (sign_a >= 0) ? result : 0 <=> result;
```
(obviously non-blocker)

Reply:
Or a local lambda, for the first part to make it cleaner:
```C++
    auto sign = [](int64_t x) { return x == 0 ? 0 : x < 0 ? -1 : 1; };
    int sign_a = sign(a1) * sign(a2);
    int sign_b = sign(b1) * sign(b2);
    if (sign_a != sign_b) return sign_a <=> sign_b;
```